docs/conf.py

```diff
@@ -225,7 +225,7 @@
 # the docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
 # pixels large.
 #
-# html_favicon = None
+html_favicon = "images/favicon.ico"
```

Comment:
We have a favicon!

Reply:
Note the new favicon on the left most tab and the RTD icon on the right most tab:

<img width="575" alt="screenshot 2019-02-07 22 22 02" src="https://user-images.githubusercontent.com/745966/52457250-d98a9380-2b26-11e9-89d4-b6c73b8e28e3.png">

Reply:
I'm not 100% sure this will show up on RTD but I'm hopeful.

Reply:
Loving the favicon